domains/dashingtesla.json

```diff
@@ -0,0 +1,6 @@
+{
+  "record": {
+    "CNAME": "ahmedschool25.github.io/dashingtesla.github.io"
```

Comment:
Invalid CNAME. CNAME records must be hostnames with no protocols, file paths, etc.